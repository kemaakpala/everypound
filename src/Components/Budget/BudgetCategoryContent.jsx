import Grid from '@material-ui/core/Grid';

const BudgetCategoryContent = ({children}) => {
  return (
    <Grid
      item xs={12} 
      // style={{ border: '4px solid green' }}
    >
      {children}
    </Grid>
  )
}

export default BudgetCategoryContent;