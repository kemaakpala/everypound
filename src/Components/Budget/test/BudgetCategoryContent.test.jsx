import BudgetCategoryContent from '../BudgetCategoryContent';
import { render } from '@testing-library/react';

describe('<BudgetCategoryContent />', () => {
  it('matches snapshot', () => {
    const { firstChild } = render(<div><BudgetCategoryContent /></div>)
    expect(firstChild).toMatchSnapshot();
  })
})