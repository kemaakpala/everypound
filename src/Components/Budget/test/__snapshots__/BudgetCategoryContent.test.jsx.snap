// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BudgetCategoryContent /> matches snapshot 1`] = `undefined`;
